.eslintignore

```diff
@@ -1,3 +1,4 @@
+lib/internal/v8.js
```

Comment:
There are other `lib/internal/v8` files whitelisted. I don't think we need a comment. But either way is fine.